caniusepython3/overrides.json

```diff
@@ -6,6 +6,7 @@
     "django-social-auth": "https://pypi.python.org/pypi/python-social-auth",
     "dnspython": "https://pypi.python.org/pypi/dnspython3",
     "extras": "https://pypi.python.org/pypi/extras",
+    "flask-mail": "https://github.com/mattupstate/flask-mail/blob/master/CHANGES#L24",
```

Comment:
Can you change this to `http://pythonhosted.org/Flask-Mail/changelog.html#version-0-9-0` so the link is stable?